### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by ntnshpig**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import Layout from './components/Layout';
-import WelcomePage from './pages/WelcomePage/WelcomePage';
+import Layout from './components/Layout'
+import PageRoutes from './routes/Routes'
 
 function App() {
   return (
     <Layout>
-      <WelcomePage />
+      <PageRoutes></PageRoutes>
```
Comment (VolodymyrDevico) [inline src/App.tsx]: <PageRoutes />

File: src/components/Sidebar/DesktopSidebar/DesktopSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+              </SidebarLinkStyled>
+            )
+          })}
+        </BottomMenuStackStyled>
+      </ToolbarStyled>
+    </SidebarBoxStyled>
+  )
+}
+
+export default DesktopSidebar
```
Comment (VolodymyrDevico) [inline src/components/Sidebar/DesktopSidebar/DesktopSidebar.tsx]: memo(DesktopSidebar)

File: src/components/Sidebar/MobileSidebar/MobileSidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <CloseIcon />
+          </MenuCloseButtonStyled>
+        </Stack>
+        <SidebarBoxStyled p={2} role="presentation">
+          <LogoBoxStyled>
+            <h3>LOGO</h3>
+          </LogoBoxStyled>
+          <ToolbarStyled>
+            <TopMenuStackStyled>
+              {upSidebarLinks.map(item => {
```
Comment (VolodymyrDevico) [inline src/components/Sidebar/MobileSidebar/MobileSidebar.tsx]: item =>(
                  <SidebarButtonStyled {...scrollConfig} to={item.link}>
                    {item.icon}
                    {item.title}
                  </SidebarButtonStyled>
                )

File: src/components/Sidebar/MobileSidebar/MobileSidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                return (
+                  <SidebarButtonStyled {...scrollConfig} to={item.link}>
```
Comment (VolodymyrDevico) [inline src/components/Sidebar/MobileSidebar/MobileSidebar.tsx]: key!

File: src/components/Sidebar/SidebarLinks.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    {
+        icon: <WorkspacesIcon />,
+        title: "Partners",
+        link: "partners"
+    },
+]
+
+export const downSidebarLinks = [
+    {
+        icon: <BusinessOutlinedIcon />,
```
Comment (VolodymyrDevico) [inline src/components/Sidebar/SidebarLinks.tsx]: Icon: BusinessOutlinedIcon
<Icon />

**Review by VolodymyrDevico — CHANGES_REQUESTED** [review]

**Review by turchin-aa — COMMENTED** [review]
This looks good!

**Review by cstruchevs — APPROVED** [review]

**Pull request merged**